pages/advanced-algorithms/available-algorithms/algo.mdx

```diff
@@ -46,6 +46,8 @@ filters.
 - `end_node: Node` ➡ The final node of the returned path.
 - `relationship_types: List[String]` ➡ A list of relationship filters, explained below.
 - `max_length: int` ➡ Max path length.
+- `subgraph: Graph` ➡ A specific subgraph, which is an object of type Graph returned by the `project()` function, on which the algorithm is run.
```

Comment:
When using subgraph as an input, it has to be used as a first argument of the procedure. I would move it to first place in this list of inputs and add a note that it's an optional argument